addons/website/__manifest__.py

```diff
@@ -112,7 +113,6 @@
         'data/website_demo.xml',
     ],
     'application': True,
-    'post_init_hook': 'post_init_hook',
```

Comment:
In the end, maybe don't remove this, since the function does nothing now (and therefore don't write "remove in master" in the function)
I think it could be useful to keep it so we know it exists :slightly_smiling_face: (in 17.0 for example, it does other things in addition to the update of images, so it might be useful to keep it)

Reply:
Indeed, and it even won't be removed in master because it does another thing as well.